ge06_team03/data/serial_number_data.xml

```diff
@@ -5,7 +5,7 @@
             <field name="name">Serial Number</field>
             <field name="code">serial.number</field>
             <field name="prefix"></field>
-            <field name="padding">6</field>
```

Comment:
if you are gonna edit a record then do it this way https://github.com/odoo-ps/psus-training/pull/181/files#diff-22e03703bd74629fe6c4b1c65ec6f6c6c9bc6693d8c24c459a1a22a318d127e9